pkg/operator/render.go

```diff
@@ -91,7 +91,6 @@ func createDiscoveredControllerConfigSpec(infra *configv1.Infrastructure, networ
 		CloudProviderConfig: "",
 		EtcdDiscoveryDomain: infra.Status.EtcdDiscoveryDomain,
 		Platform:            platform,
-		Proxy:               &proxy.Status,
```

Comment:
Stray change from the other PR here?

Reply:
yup, but didn't want to rebase since this won't be a conflict once it merges I guess 🤔